Classes/YIPopupTextView.m

```diff
@@ -545,18 +546,15 @@ - (void)_changePopupViewFrameWithNotification:(NSNotification*)notification
     
     CGFloat topMargin = _topUIBarMargin;
     CGFloat bottomMargin = _bottomUIBarMargin;
-    
     CGFloat popupViewHeight = 0;
     
-#if defined(__IPHONE_7_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_7_0
-    
-    // automatically adjusts top/bottomUIBarMargin for iOS7 fullscreen size
+
     if (_viewController) {
         UINavigationBar* navBar = _viewController.navigationController.navigationBar;
         UIToolbar* toolbar = _viewController.navigationController.toolbar;
         UITabBar* tabBar = _viewController.tabBarController.tabBar;
         
-        CGFloat statusBarHeight = (IS_PORTRAIT ? [UIApplication sharedApplication].statusBarFrame.size.height : [UIApplication sharedApplication].statusBarFrame.size.width);
```

Comment:
To be iOS7 Compatible, there only need be a check on this line, to return:
```objective-c
 [UIApplication sharedApplication].statusBarFrame.size.width
```
However, didn't add because I think iOS 8 is the requirement now.